.github/workflows/tpv2.yml

```diff
@@ -26,12 +26,18 @@ on:
       - experimental/traffic-portal/**
       - .github/workflows/tpv2.yml
       - .github/actions/tpv2-integration-tests
+      - lib/go-tc/**
```

Comment:
I think the reason we weren't doing these in TPv2 is because it's still considered experimental and as such didn't want them running on/holding up Go PRs.  Perhaps that is no longer the case and we should update the file structure in someway (and the CHANGELOG probably), but if it is still experimental, then we don't want this.

Reply:
>  Perhaps that is no longer the case and we should update the file structure in someway (and the CHANGELOG probably)

Right, I think that's no longer the case. *Experimental* implies people shouldn't try to address serious use-cases with it, so maybe more accurately it could be called Traffic Portal Alpha or something. At some point, we should just call it Traffic Portal and rename the old one to something like Traffic Portal.js